src/main/java/seedu/address/logic/Messages.java

```diff
@@ -40,11 +40,12 @@ public static String format(Person person) {
                 .append("; Phone: ")
                 .append(person.getPhone())
                 .append("; Email: ")
-                .append(person.getEmail())
-                .append("; Address: ")
-                .append(person.getAddress())
-                .append("; Tags: ");
-        person.getTags().forEach(builder::append);
+                .append(person.getEmail());
+        if (person.getAddress() != null) {
+            builder.append("; Address: ").append(person.getAddress());
+        }
```

Comment:
i think it might be nicer to use `ifPresent()` to avoid comparing with `null`

something similar to `Person.toString()`